sufia.gemspec

```diff
@@ -9,12 +9,14 @@ Gem::Specification.new do |gem|
   gem.homepage      = "http://github.com/projecthydra/sufia"
 
   gem.files         = `git ls-files | grep -v ^sufia-models`.split($\)
+  gem.bindir        = 'exe'
   gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = "sufia"
   gem.require_paths = ["lib"]
   gem.version       = version
   gem.license       = 'Apache2'
+  gem.executables   = gem.files.grep(%r{^exe/}) { |f| File.basename(f) }
```

Comment:
I was following the location of the executables, but now that you point out line 13 I will move my executable to bin.